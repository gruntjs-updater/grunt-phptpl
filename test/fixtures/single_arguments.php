<?php 
echo $argv[1]."\nSTATIC\n".$argv[2] ;