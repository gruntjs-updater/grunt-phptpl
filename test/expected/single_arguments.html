ARGVALUE1!
STATIC
ARGVALUE2!